sci-calculators/qalculator/qalculator-0.4b-r1.ebuild

```diff
@@ -18,6 +18,10 @@ DEPEND="dev-qt/qtgui:4"
 
```

Comment:
Have you checked that `qtgui` is really just a build-time dependency? My gut feeling tells me this doesn't seem right.